pkg/network/usm/ebpf_main.go

```diff
@@ -359,6 +353,10 @@ func (e *ebpfProgram) init(buf bytecode.AssetReader, options manager.Options) er
 			log.Debugf("disabled map: %v", m.Name)
 		}
 
+		for _, probe := range p.Probes {
```

Comment:
Why we need to split probe from tail call in this context? it's doing the same logic. I would suggest to rename `TailCalls` to `Probes` and add all relevant probes there

Reply:
you can't as we need to distinguish between probes and tail calls
https://github.com/DataDog/datadog-agent/blob/73867119465d5e492d7674f222172bd800108842/pkg/network/usm/monitor.go#L305